modules/zivid/matrix4x4.py

```diff
@@ -0,0 +1,93 @@
+"""Contains Matrix4x4 class."""
+import pathlib
+import _zivid
+
+
+class Matrix4x4(_zivid.Matrix4x4):
+    """Matrix of size 4x4 containing 32-bit floats."""
+
+    def inverse(self):
+        """
+        Return the inverse of this matrix.
+
+        An exception is thrown if the matrix is not invertible.
+
+        Returns:
+            A new matrix, holding the inverse
+        """
+        return Matrix4x4(super().inverse())
+
+    def load(self, file_path):
+        """
+        Load the matrix from the given file.
+
+        Args:
+            file_path: path for the file to load the matrix from.
+        """
+        if isinstance(file_path, pathlib.Path):
+            file_path = str(file_path)
+        super().load(file_path)
+
+    def save(self, file_path):
+        """
+        Save the matrix to the given file.
+
+        Args:
+            file_path: path for the new file to save the matrix in.
+        """
+        if isinstance(file_path, pathlib.Path):
+            file_path = str(file_path)
+        super().save(file_path)
+
+    def __getitem__(self, indexes):
+        """
+        Access specified element with bounds checking.
+
+        Args:
+            indexes: a tuple of 2 integers as the indexes.
+
+        Returns:
+            The accessed item.
+        """
+        return super()._getitem(indexes)
+
+    def __init__(self, arg=None):
+        """
+        Overloaded constructor.
+
+        Does different kinds of initializations depending on the argument:
+
+        * None or no arguments -> Zero initializes all values.
+        * List[float[16]] -> Map 1D array of 16 values into this 2D array of 4x4.
+        * List[List[float[4]][4]] -> Copy 2D array of size 4x4.
```

Comment:
This can also be numpy arrays, right?

Reply:
Yes, definitely. A bit harder to word in here. How about:

```
* ndarray/list of 16 floats -> ...
* ndarray/list of 4x4 float matrix -> ...
```

Reply:
Yeah I like that.